<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QMainWindow" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDoubleSpinBox" name="CreditSumm">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Month">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>600.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string> Сумма кредита</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Процентная ставка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Срок кредита /мес.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MonthPay">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="OverPay">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="TotalPay">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: #ffffff;
    font: bold 14pt &quot;Arial&quot;;
    color: rgb(126, 126, 123);
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Ежемесячный платеж</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: #ffffff;
    font: bold 14pt &quot;Arial&quot;;
    color: rgb(126, 126, 123);
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Переплата по кредиту</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: #ffffff;
    font: bold 14pt &quot;Arial&quot;;
    color: rgb(126, 126, 123);
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Общая выплата</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Диференциальный</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: rgb(126, 126, 123);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Ануететный (по умолчанию)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(108, 210, 96);
    font: bold 14pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}
QPushButton:pressed {
	background-color: rgb(242, 242, 242);
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(242, 158, 51);
    font: bold 18pt &quot;Arial&quot;;
    color: #ffffff;
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}

QPushButton:pressed {
	background-color: rgb(242, 242, 242);
	border: 1px solid rgb(82, 83, 79);
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string> Назад / Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
